modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusStateUpdater.scala

```diff
@@ -269,7 +270,8 @@ object ConsensusStateUpdater {
                     state.lastOutcome.status.context,
                     majorityTrigger,
                     resources,
-                    scoredProposalHashes
+                    scoredProposalHashes,
+                    candidates
```

Comment:
Why `candidates` rather than `facilitators`?

Reply:
It has been already named `candidates` in `ConsensusStateUpdater`by Tomek. I just passed it to the function.

Reply:
I see that it's passed and I think it may be the wrong value, didn't you mean `state.facilitators` here

Reply:
Good catch as `candidates` is just a diff for nodes we should add to the consensus can be empty. I will change it to `state.facilitators`.